charts/agent-control/templates/install-flux.yaml

```diff
@@ -28,6 +28,10 @@ spec:
               cpu: 50m
               memory: 64Mi
           command: ["/bin/sh", "-c"]
+          envFrom:
+            - secretRef:
+                name: {{ .Values.installationJob.agentControlCd.repositorySecretReferenceName }}
+                optional: true
```

Comment:
why not leveraging {{ if - }} here instead of marking it as optional?

I fear it could create confugsion

Reply:
Good catch! I think it's better. Changed in https://github.com/newrelic/helm-charts/pull/1860/commits/769eb37ee43f29900dd0d00ae0700213ff091fd5!